client/app/shared/select/select.component.html

```diff
@@ -0,0 +1,4 @@
+<div><app-label [compConfig]="labelConfig"></app-label></div>
```

Comment:
Id should be unique for every instance of this component.

Name should be generic